docs/source/aws/policy/lambda.rst

```diff
@@ -193,6 +193,39 @@ EC2 instance with the ``Custodian`` tag, and stops it according to your policy.
 Congratulations! You have now installed your policy to run under Config rather
 than from your command line.
 
+Lambda Configuration
+####################
+
+Custodian lambdas support configuring all lambda options via keys on the lambda
+mode in the YAML.  See AWS'
+`AWS Lambda Function Configuration <https://docs.aws.amazon.com/lambda/latest/dg/resource-model.html>`_
+page for the full list of configuration options avaible on a Lambda.
+
```

Comment:
it would be nicer to add a reference to using `custodian schema mode.periodic` as we're just covering common config here, where was that shows the full configuration options for each of the modes.

Reply:
actually Austen just added in some mode documentation that references the full schema so we can skip that here, but it would be good

Reply:
Added in a reference to the AWS modes section